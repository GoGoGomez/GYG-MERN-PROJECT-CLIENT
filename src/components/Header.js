import React from 'react';
import { NavLink } from 'react-router-dom';
// import Nav from '../styles/component/header.css'

const Header = () => (
  <header className="Header">
      <div className="Header-title">GyG Kawana</div>
    <div className="Header-links">
      <NavLink to="/" activeClassName="is-active" exact={true}>Home</NavLink>
      <NavLink to="/menu" activeClassName="is-active" exact={true}>Menu</NavLink>
      <NavLink to="/checkout" activeClassName="is-active" exact={true}>Checkout</NavLink>
    </div>
  </header>
);

export default Header;