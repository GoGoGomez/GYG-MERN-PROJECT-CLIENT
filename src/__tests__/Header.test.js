import React from "react";
import renderer from "react-test-renderer";
import Header from "../components/Header";
import { MemoryRouter as Router, withRouter } from "react-router-dom";

test("Header renders correctly", () => {
  const component = renderer.create(<Router><Header /></Router>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
