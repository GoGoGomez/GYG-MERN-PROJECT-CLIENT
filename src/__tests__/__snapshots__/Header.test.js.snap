// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="Header"
>
  <div
    className="Header-title"
  >
    GyG Kawana
  </div>
  <div
    className="Header-links"
  >
    <a
      aria-current="page"
      className="is-active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Home
    </a>
    <a
      aria-current={null}
      className={undefined}
      href="/menu"
      onClick={[Function]}
      style={undefined}
    >
      Menu
    </a>
    <a
      aria-current={null}
      className={undefined}
      href="/checkout"
      onClick={[Function]}
      style={undefined}
    >
      Checkout
       
    </a>
  </div>
</header>
`;
